reana_workflow_engine_yadage/tasks.py

```diff
@@ -26,22 +26,22 @@
 import logging
 import os
 
-import pika
 import zmq
 from yadage.steering_api import steering_ctx
 from yadage.utils import setupbackend_fromstring
 
 from . import celery_zeromq
 from .celeryapp import app
-from .config import (BROKER_PASS, BROKER_PORT, BROKER_URL, BROKER_USER,
-                     CODE_DIRECTORY_RELATIVE_PATH,
+from .config import (CODE_DIRECTORY_RELATIVE_PATH,
                      INPUTS_DIRECTORY_RELATIVE_PATH,
                      LOGS_DIRECTORY_RELATIVE_PATH,
                      OUTPUTS_DIRECTORY_RELATIVE_PATH, SHARED_VOLUME_PATH,
                      YADAGE_INPUTS_DIRECTORY_RELATIVE_PATH)
+from .utils import publish_workflow_status
```

Comment:
@dinosk missing `utils.py`? I guess that this function will end up in commons?

Reply:
@diegodelemos Right I'm adding it. About moving it to commons, I think we don't want to add the dependency to it in the workflow engines.